### ISSUE TITLE
scala-steward doesn't exit cleanly if required options not present

### ISSUE THREAD

**Issue by anilkumarmyla** [fthomas/scala-steward#411]
Compiled and running on jdk11

```
$ bin/scala-steward 
java.lang.Throwable: Required option --workspace not specified
Required option --repos-file not specified
Required option --git-author-name not specified
Required option --git-author-email not specified
Required option --github-login not specified
Required option --git-ask-pass not specified
	at org.scalasteward.core.application.Cli$$anon$1.$anonfun$parseArgs$6(Cli.scala:70)
	at cats.syntax.EitherOps$.bimap$extension(either.scala:128)
	at org.scalasteward.core.application.Cli$$anon$1.parseArgs(Cli.scala:70)
	at org.scalasteward.core.application.Context$.create(Context.scala:54)
	at org.scalasteward.core.steward$.run(steward.scala:28)
	at cats.effect.IOApp.$anonfun$main$3(IOApp.scala:67)
	at cats.effect.internals.IOAppPlatform$.mainFiber(IOAppPlatform.scala:36)
	at cats.effect.internals.IOAppPlatform$.main(IOAppPlatform.scala:24)
	at cats.effect.IOApp.main(IOApp.scala:67)
	at cats.effect.IOApp.main$(IOApp.scala:66)
	at org.scalasteward.core.steward$.main(steward.scala:26)
	at org.scalasteward.core.steward.main(steward.scala)
^C^C^D
```

Any key combination after this doesn't exit the program

**Comment by mcanlas** [comment]
I've run into this several times myself. I'm not sure what the root problem is but I did create a minimal example at https://github.com/typelevel/cats/issues/2805.

**Comment by mcanlas** [comment]
TLDR it's a bug in `cats-effect`. I think it should be fixed seamlessly with a (pending) upgrade.

**Comment by anilkumarmyla** [comment]
cats-effect-1.3.0 is out 😋

**Comment by fthomas** [comment]
scala-steward will update itself soon :-)

**Comment by anilkumarmyla** [comment]
verified that #459 fixes this
